src/utils/utils/units.ts

```diff
@@ -67,3 +53,13 @@ export const readableSizeUnit = (combinedStr: string): string => {
   // if there isn't any specific value/size present, return the original string, for example for the dynamic disk size
   return !value ? combinedStr : `${value} ${toIECUnit(unit)}`;
 };
+
+export const getHumanizedSize = (size: string, bytesOption: 'withB' | 'withoutB' = 'withB') => {
+  const baseValue = convertToBaseValue(size);
```

Comment:
it's out-of-scope of this PR but note this step involves `parseFloat` and will fail on localized numbers (a follow-up?).